examples/app_commands/basic.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+class MyBot(commands.Bot):
+
+    # In this basic example, we just synchronize the app commands to one guild,
+    # without requiring to set the guild to every command individually by just copying them over.
+    # By doing so we don't have to wait up to 1hr until they are shown for the end-user.
+
+    # If you want to synchronize your app commands to every guild your bot is in, 
+    # remove the following setup_hook and uncomment the one below.
+    async def setup_hook(self):
+        # Replace ... with the guild id you want your commands to be synced with.
+        guild = discord.Object(...)
```

Comment:
It might be better to make this a global constant, e.g. `MY_GUILD = discord.Object(id=0)  # replace with your ID here` to steer people into the right direction.